pages/whats-new/product-roadmap.md

```diff
@@ -11,96 +11,89 @@ redirect_from:
   - /whats-new/product-roadmap/
 ---
 
-Below is our product roadmap: a long-term plan of the goals, features, and long-term direction of USWDS. We update this frequently with the status of our progress, as well as add new high-level feature requests and ideas.
+Here, you’ll find our product roadmap — an up-to-date report on the work we’re doing.
+
+This roadmap communicates what we've recently done and what we've got planned in the next 4-8 months. We update this page every 4 months (our roadmap planning increments). 
+
+Check out our <a href="https://github.com/orgs/uswds/projects/8/views/1">Public GitHub Roadmap</a>for more details about our work.
```

Comment:
Just confirming. Is the leaderboard tab of the new project board the correct link?

Reply:
I believe so, but @elisaachen can confirm

Reply:
> Just confirming. Is the leaderboard tab of the new project board the correct link?

It should be pointing to our new public roadmap board here https://github.com/orgs/uswds/projects/13/views/1

Reply:
I've updated to the correct link